src/main/java/com/conveyal/r5/profile/ProfileRequest.java

```diff
@@ -236,6 +237,7 @@ This parameter compensates for the fact that GTFS does not contain information a
 
     //If true current search is reverse search AKA we are looking for a path from destination to origin in reverse
     //It differs from searchType because it is used as egress search
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
```

Comment:
Is this change related to worker startup? If not can it be separated out of the PR?